content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+
+In this example, a browser test is configured to map a user's journey from adding an item to cart to successful checkout. If any step during the browser test fails, it throws an error that is recorded in Datadog as a **Test Result**.
+
+{{< img src="getting_started/synthetics/browser-test.png" alt="Browser test" responsive="true" style="width:90%;" >}}
+
+#### Configure your test
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **Browser Test**.
+4. Define the configuration of your browser test:
+    - Add a URL. In this example the URL is https://www.shopist.io/.
+    - Name the test.
+    - Set tags to help organize tests.
+    - Choose devices and locations for testing. In this example, the test is only run on **Large Laptop** & on **English speaking countries**.
+    - Specify a test frequency.
+    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert. To avoid being alerted on network blips that might happen on specific locations, this test is configured as:
+        `An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations`
+        `Retry 1 time before location is marked as failed`
+    - Write an alert message and specify who to notify when the alert is triggered.
+    - Click **Save & Edit Recording**.
+
+{{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test" responsive="true" style="width:90%;">}}
+
+#### Record your test steps
+
+Once the test configuration is saved, Datadog will prompt you to download the [Datadog test recorder][4] extension. Browser tests can be only recorded on **[Google Chrome][5]**. Download and install the extension.
+
+Once this extension is installed, begin recording your test steps by clicking the **Start Recording** button. Navigate your page in the iframe to the right of the recording options. When you select a div, image, or any area of your page, the actions are recorded and used to create steps within the browser test. You can record the uploading of files as an action, though this is limited to 10 files, with a limit of 5MB each. 
+
+For example, to record test steps that map a user's journey from adding an item to cart to successful checkout:
+
+1. Navigate to one of the furniture sections, for instance **Chairs**, and select **Add to cart**.
+2. Click on **Cart**, click **Checkout**.
+3. Add the **assertion** “Test text is present on the active page” to confirm the words “Thank you” are on the page. 
+    **Note**: Your last browser test step must be an **assertion**. This will ensure your test ended up on the expected page and found the expected element. 
+4. Save the test.
+
+{{< img src="getting_started/synthetics/record-test.gif" alt="Record test steps" responsive="true" style="width:90%;">}}
```

Comment:
Can we use that gif https://cl.ly/0116a5aedd01 instead? `Laptop large` should be selected in the dropdown (otherwise it might cause users' tests to fail), and let's not show any account data

Reply:
Added gif suggestion.